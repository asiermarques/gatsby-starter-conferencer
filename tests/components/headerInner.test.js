import React from "react"
import renderer from "react-test-renderer"
import Header from "../../src/components/headerInner"
import HeaderConfigFixture from "../__fixtures/HeaderConfig"

describe("HeaderInner", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Header  data={HeaderConfigFixture}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})