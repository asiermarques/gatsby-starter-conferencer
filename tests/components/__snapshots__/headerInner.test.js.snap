// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderInner renders correctly 1`] = `
<div
  class="jumbotron"
>
  <div
    class="container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-7"
      >
        <a
          href="/"
        >
          <h2
            class="display-4"
          >
            Our awesome conference
          </h2>
        </a>
        <p
          class="lead"
        >
          A great conference for developers and other mystical creatures
        </p>
      </div>
      <div
        className="col-sm-5 text-center"
      >
        <p>
          100 early-bird tickets left
        </p>
        <a
          class="btn btn-primary btn-lg"
          href="https://awesome_url_for_your_ticketing_platform.com"
          role="button"
        >
          Tickets on sale!
        </a>
      </div>
    </div>
  </div>
</div>
`;
